docs/news/posts/2024-10-18-browsing-curated-database.md

```diff
@@ -2,56 +2,120 @@
-The final goal will be to create a table of the value of the band gap for a set of 2D materials from this study, filtering only those where the band gap is below a given value.
-We will show you here, step by step, how to start from a new unknown database, understand its layout, and eventually prepare the final query.
-This will give you some guidance on how to apply and extend queries for your own needs.
+A common source of problems for new users of AiiDA often arises when they receive an exported database (archive) and want to get an idea of its content but don't know how.
 
-This story can be run in the [Quantum Mobile 20.03.0](https://github.com/marvel-nccr/quantum-mobile/releases/tag/20.03.0) virtual machine, that you need to download and install first (unless you have AiiDA already installed on your computer; we are using AiiDA 1.3 here).
-You will also need to download and import the [2D materials database](https://archive.materialscloud.org/record/file?filename=two_dimensional_database.aiida&file_id=d1f3ac29-e3b0-400b-8109-8455be66160b&record_id=18) from [the corresponding Materials Cloud Archive entry](https://archive.materialscloud.org/record/2017.0008/v3).
+In this blog post we will use the database from ["Two-dimensional materials from high-throughput computational exfoliation of experimentally known compounds"](_CHANGE_).
+The description of the database mentions that it contains a set of two dimensional materials, and for a subset of them, various properties are calculated, including the electronic band structures (and also the corresponding band gaps).
+
+To demonstrate how to start from an unknown database, understand its layout, and query for a particular property, we set a goal for the blog post to filter the materials based on their band gaps and create a corresponding table.
+This will give you guidance on how to apply and extend queries for your own needs.
+
+Note that while this is a curated database, meaning that it has supplementary metadata provided to help understand it, most of this blog post also applies to uncurated AiiDA archives.
+
+## Setup
+
+This analysis can be run with AiiDA v2+. If it is not installed, the easiest option is to use [Quantum Mobile 24.04.0](https://quantum-mobile.readthedocs.io/en/latest/releases/versions/24.04.0.html) virtual machine, which was used for this blog post.
+You will also need to download and import the AiiDA archive `two_dimensional_database.aiida` from [the corresponding Materials Cloud Archive entry](_CHANGE_).
 This can be easily done by running the following commands in your AiiDA environment:
 
 ```bash
-(aiida) max@qmobile:~$ wget 'https://archive.materialscloud.org/record/file?file_id=d1f3ac29-e3b0-400b-8109-8455be66160b&filename=two_dimensional_database.aiida&record_id=18' -O 2D_database.aiida
-(aiida) max@qmobile:~$ verdi import 2D_database.aiida
+(aiida) max@qmobile:~$ wget _CHANGE_
+(aiida) max@qmobile:~$ verdi archive import two_dimensional_database.aiida
```

Comment:
Just wondering if we should now provide the alternative for people that don't have a profile yet, or don't want to pollute it. They can create a new profile from the archive with `verdi profile setup core.sqlite_zip --filepath two_dimensional_database.aiida`

Reply:
thanks @sphuber, great point. Sadly not yet available in Quantum mobile but i will add this option either way.